README.adoc

```diff
@@ -335,41 +321,41 @@ installation. The type of server package is configured in your `pom.xml` file in
 <include>${packaging.type}</include>
 ----
 
-Alternatively to creating a server package, you can generate a runnable JAR file that contains the
-application along with a server runtime. This JAR can then be run anywhere, deploying your application
-and server in one go. To do this, run the following command:
+Instead of creating a server package, you can generate a runnable JAR file that contains the
+application along with a server runtime. This JAR can then be run anywhere and deploy your application
+and server at the same time. To generate the JAR file, run the `install -P runnable-package` command:
 
 ```
 mvn install -P runnable-package
 ```
 
-The `-P` flag specifies a Maven profile to be run during the build. In this case, the `runnable-package`
-profile is invoked, which temporarily overrides the `packaging.type` property from `usr` to `runnable`.
-This property then propagates to the `liberty-maven-plugin` which generates the desired server package:
+The `-P` flag specifies the Maven profile to run during the build. In this case, the `runnable-package`
+profile is invoked, which temporarily overrides the `packaging.type` property from the `usr` package to the `runnable` package.
+This property then propagates to the `liberty-maven-plugin` plug-in, which generates the required server package:
 
 [source,xml,indent=0, role="no_copy"]
 ----
 include::finish/pom.xml[tags=profile-runnable-package]
 ----
 
-When the build completes, you can find the runnable `getting-started.jar` file under the `target`
-directory. By default, this JAR comes with all the features available in Open Liberty, which include
+When the build completes, you can find the runnable `getting-started.jar` file in the `target`
+directory. By default, this JAR file comes with all the features available in Open Liberty, including
 the entirety of Java EE 7 and MicroProfile 1.3. As a result, this JAR is about 93 MB. To omit
 the features that you don't need and package the JAR with only the features that you
-defined in the `server.xml` file, specify the packaging type to be `minifiy,runnable`. We've already
-configured the `minify-runnable-package` profile to do this for you. Invoke the profile as you did
-earlier by using the `-P` flag:
+defined in the `server.xml` file, specify the `minifiy,runnable` packaging type.
+The `minify-runnable-package` profile is already configured to package the JAR with only the features you want. Invoke the profile
```

Comment:
Maybe "The `minify-runnable-package` profile is already configured to produce a minimal runnable JAR".

Reply:
Yes, your suggestion sounds good. :)